src/Raven.Server/Documents/ETL/EtlLoader.cs

```diff
@@ -19,7 +22,7 @@
 
 namespace Raven.Server.Documents.ETL
 {
-    public class EtlLoader : IDisposable
+    public class EtlLoader : IDisposable, ITombstoneAware
```

Comment:
Why do we need to make the loaded aware of tomstones? Why was it not enough that each process implemented `ITombstoneAware`?

Reply:
Because even node that isn't responsible for the ETL should take it into account.
It can't purge the tombstone until the responsible  node sent it successfully.

Reply:
Oh, I see it now. Thanks